package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "dependencies": {
```

Comment:
is this still used since you installed `nouislider` and `bootstrap` by adding the files or using a CDN? Also, the node-modules is in gitignore.
Same goes for `package-lock.json`